bundle/manifests/collector_monitoring.coreos.com_v1_prometheusrule.yaml

```diff
@@ -117,15 +105,12 @@ spec:
         severity: Warning
   - name: logging_clusterlogging_telemetry.rules
     rules:
-    - expr: |
```

Comment:
is this only a recording rule from the fluent metrics?  I thought this was in place to normalize some of our queries so they were collector agnostic

Reply:
Yes this rule used only for Fluentd, but for Vector we have vector_component_received_bytes_total(24h)